src/utils/money.cljs

```diff
@@ -239,9 +239,12 @@
 
 (defn crypto->fiat
   [crypto fiat-price]
-  (when-let [^js bn (bignumber crypto)]
-    (-> (.times bn ^js (bignumber fiat-price))
-        (with-precision 2))))
+  (let [^js crypto-bn     (bignumber crypto)
+        ^js fiat-price-bn (bignumber fiat-price)]
+    (when (and crypto-bn fiat-price-bn)
+      (-> crypto-bn
+          (.times fiat-price-bn)
+          (with-precision 2)))))
 
```

Comment:
Now the implementation checks that both numbers are bignumbers.

I considered to use `fnil` as:

```clojure
(fnil bignumber 0)
```

But I was afraid that it broke existing usages where we relied on:

```clojure
(bignumber nil) ;; => nil
```